packages/vue-client/src/views/RulesView.vue

```diff
@@ -27,9 +27,7 @@ function toUpperCaseFirstLetter(string: string) {
 .rules-page {
   font-family: "helvetica", "arial", "sans-serif";
 }
-strong,
-h1,
-h2 {
+:deep(strong, h1, h2) {
```

Comment:
I'm not familiar with :deep, is this a vue thing?

Reply:
Yes I believe so: https://vuejs.org/api/sfc-css-features.html#deep-selectors

As I understand, it makes it so the css selector is applied "outside of scope" to this components child components.

The hint on this site is exactly the case for us:
`DOM content created with v-html are not affected by scoped styles, but you can still style them using deep selectors`

Reply:
Cool, this seems useful.  Thanks for the pointer to the docs!